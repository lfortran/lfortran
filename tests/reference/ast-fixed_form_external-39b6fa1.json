{
    "basename": "ast-fixed_form_external-39b6fa1",
    "cmd": "lfortran --indent --fixed-form --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/fixed_form_external.f",
    "infile_hash": "0308bf93ab11c4ed94717a33b3b27786a7103f661df91d7613142abb",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-fixed_form_external-39b6fa1.stdout",
    "stdout_hash": "4201117b68d5a2159a340fb8e90b00dfa10345f7e7f4b9ec52bd058f",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}