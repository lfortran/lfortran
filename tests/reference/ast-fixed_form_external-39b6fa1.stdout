(TranslationUnit 
    [(Program 
        main 
        () 
        [] 
        [] 
        [(Declaration 
            () 
            [(SimpleAttribute 
                AttrExternal
            )] 
            [(f 
            [] 
            [] 
            () 
            None 
            ())] 
            ()
        )
        (Declaration 
            (AttrType 
                TypeInteger 
                [] 
                () 
                None
            ) 
            [] 
            [(b 
            [] 
            [] 
            () 
            None 
            ())
            (c 
            [] 
            [] 
            () 
            None 
            ())
            (d 
            [] 
            [] 
            () 
            None 
            ())] 
            ()
        )] 
        [(= 
            0 
            b 
            1 
            ()
        )
        (SubroutineCall 
            0 
            f 
            [] 
            [(() 
            b 
            ())] 
            [] 
            ()
        )] 
        []
    )
    (Subroutine 
        f 
        [(a)] 
        [] 
        () 
        () 
        [] 
        [] 
        [] 
        [(Declaration 
            (AttrType 
                TypeInteger 
                [] 
                () 
                None
            ) 
            [] 
            [(a 
            [] 
            [] 
            () 
            None 
            ())] 
            ()
        )] 
        [(Print 
            0 
            () 
            [a] 
            ()
        )] 
        []
    )]
)
